### ISSUE TITLE
if token limit reached in OpenAI API, client hangs

### ISSUE THREAD

**Issue by pseudotensor** [vllm-project/vllm#1087]
Related: https://github.com/vllm-project/vllm/issues/525

Repro:
* Start vllm, e.g.

```bash
export NCCL_IGNORE_DISABLED_P2P=1
export CUDA_VISIBLE_DEVICESs=0,1,2,3 
python -m vllm.entrypoints.openai.api_server --port=5000 --host=0.0.0.0 --model h2oai/h2ogpt-4096-llama2-70b-chat --tokenizer=hf-internal-testing/llama-tokenizer --tensor-parallel-size=4 --seed 1234
```

* Connect via OpenAI API and make prompt too large (> 2560).

**Comment by supersat** [comment]
This appears to be fixed as of v0.2.0

**Comment by supersat** [comment]
In particular, it appears to have been fixed in #1029

**Comment by pseudotensor** [comment]
I'll close assuming fixed
